cloudbridge/cloud/base/resources.py

```diff
@@ -803,7 +831,7 @@ def __repr__(self):
 class BaseNetwork(BaseCloudResource, BaseObjectLifeCycleMixin, Network):
 
     CB_DEFAULT_NETWORK_NAME = os.environ.get('CB_DEFAULT_NETWORK_NAME',
-                                             'CloudBridgeNet')
+                                             'cloudbridge_net')
```

Comment:
How about swapping the underscores here to dashes for improved url structure in cases this is used as a resource ID?

Reply:
@afgane Are you thinking cloud-bridge-net, or cloudbridge-net?

Reply:
`cloudbridge-net`

Reply:
Thanks for pointing this out. Fixed in: https://mail.google.com/mail/u/0/#inbox/15e5e14123de6359

Reply:
Thanks for pointing this out. Fixed in: https://github.com/gvlproject/cloudbridge/commit/bccac478d8a16f249d34a4cd2b0c87bd24124416